kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -218,8 +218,8 @@ define([
             });
         }
 
-        function addRow(){
-            $(`.${cssBaseClass}__table`).append(
+        function addRow(e){
+            $(e.target).prev('table').append(
```

Comment:
@briehl This was causing the row issue. The selector returned all tables. So when there was more than one table in the narrative, a row was added to each.